src/Core/Domain/Shop/Command/UploadLogosCommand.php

```diff
@@ -126,4 +140,22 @@ public function setUploadedFavicon(UploadedFile $uploadedFavicon)
 
         $this->uploadedFavicon = $uploadedFavicon;
     }
+
+    /**
+     * @param UploadedFile $uploadedFile
+     *
+     * @throws NotSupportedLogoImageExtensionException
+     */
+    private function assertIsValidLogoImageExtension(UploadedFile $uploadedFile)
```

Comment:
same here, should the extra validation be part of the command responsibility?

Reply:
For an extension I think it makes sense. We try to make sure you can only create valid commands. An extension is something quick to check, it's easy to identify bad values and mark the command as not valid. Wdyt ?